test/parallel/test-fs-readfile-flags.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+// Test of fs.readFile with different flags.
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+{
+  const emptyFile = path.join(tmpdir.path, 'empty.txt');
+  fs.closeSync(fs.openSync(emptyFile, 'w'));
+
+  fs.readFile(
+    emptyFile,
+    // With `a+` the file is created if it does not exist
```

Comment:
nit: this comment is misleading as `emptyFile` is already created on L14. In this test the file is just opened.